omniqueue/src/backends/redis/fallback.rs

```diff
@@ -226,7 +226,7 @@ async fn send_to_dlq<R: RedisConnection>(
         .get()
         .await
         .map_err(QueueError::generic)?
-        .lpush(dlq, payload)
+        .rpush(dlq, payload)
```

Comment:
`RPUSH` instead of `LPUSH` so that the `LRANGE` logic is more straightforward when redriving.